cmd/estuary-shuttle/rpc.go

```diff
@@ -11,53 +11,70 @@ import (
 	"github.com/filecoin-project/go-state-types/abi"
 	blocks "github.com/ipfs/go-block-format"
 	"github.com/ipfs/go-cid"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/trace"
 	"golang.org/x/xerrors"
 	"gorm.io/gorm"
 )
 
-func (d *Shuttle) handleRpcCmd(cmd *drpc.Command) error {
+func (s *Shuttle) handleRpcCmd(cmd *drpc.Command) error {
 	ctx := context.TODO()
 
+	// If the command contains a trace continue it here.
```

Comment:
Here is where the shuttle continues the trace if present in the command from Estuary.